mne/time_frequency/tfr.py

```diff
@@ -1289,22 +1303,22 @@ def _plot(self, picks=None, baseline=None, mode='mean', tmin=None,
                 yscale=yscale, mask=mask, mask_style=mask_style,
                 mask_cmap=mask_cmap, mask_alpha=mask_alpha)
 
-            if title is None:
-                if combine is None or len(tfr.info['ch_names']) == 1:
-                    title = tfr.info['ch_names'][0]
+            if title == 'auto':
+                if len(tfr.info['ch_names']) == 1:
+                    subtitle = tfr.info['ch_names'][idx]
                 else:
-                    title = _set_title_multiple_electrodes(
-                        title, combine, tfr.info["ch_names"], all=True,
-                        ch_type=ch_type)
-
-            if title:
-                fig.suptitle(title)
+                    if combine is None:
+                        subtitle = tfr.info['ch_names'][idx]
+                    else:
+                        subtitle = _set_title_multiple_electrodes(
+                            None, combine, tfr.info["ch_names"], all=True,
+                            ch_type=ch_type)
+            else:
+                subtitle = title
+            fig.suptitle(subtitle)
 
-            plt_show(show)
-            # XXX This is inside the loop, guaranteeing a single iter!
-            # Also there is no None-contingent behavior here so the docstring
-            # was wrong (saying it would be collapsed)
-            return fig
+        plt_show(show)
+        return [fig for (fig, ax) in figs_and_axes]
```

Comment:
You've created 'figs` list earlier on - you can return it here.

Reply:
Actually, earlier I created a list of (fig, axes) tuples to simplify the the indexing in the loop. So, at the return, I select only the figs, and drop the axes

Reply:
Right, the `figs` list is only present in one branch.

Reply:
The only issue might be that now you always return list of figures, while previously a single figure was returned, so the change may break code that relies on the previous behavior (plotting one channel for example). But its easy to fix so I think it should be ok.